docs/book/src/clusterctl/developers.md

```diff
@@ -56,6 +56,19 @@ in order to use them, please run:
 
 clusterctl init  --core cluster-api:v0.3.0 --bootstrap kubeadm:v0.3.0 --infrastructure aws:v0.5.0
 ```
+<aside class="note warning">
```

Comment:
being `overridesFolder` a config option, IMO it should be documented in https://cluster-api.sigs.k8s.io/clusterctl/configuration.html. Up to you if to move leave this warning here or not

Reply:
Thanks! I just wanted to document it somewhere. I'll take a look at the configuration page and see where I can add it.